packages/wdio-applitools-service/README.md

```diff
@@ -0,0 +1,72 @@
+WebdriverIO Applitools Service
+==============================
+
+> A WebdriverIO service for visual regression testing using Applitools
+
+## Installation
+
+The easiest way is to keep `wdio-applitools-service` as a devDependency in your `package.json`.
+
+```json
+{
+  "devDependencies": {
+    "wdio-applitools-service": "^5.0.0"
+  }
+}
+```
+
+You can simple do it by:
+
+```bash
+npm install wdio-applitools-service --save-dev
+```
+
+Instructions on how to install `WebdriverIO` can be found [here.](http://webdriver.io/guide/getstarted/install.html)
+
+## Configuration
+
+In order to use the service you need to set `applitoolsKey` in your `wdio.conf.js` config file or have `APPLITOOLS_KEY` stored in your environment so that it can access the Applitools API. Also make sure that you added `applitools` to your service list, e.g.
+
+```js
+// wdio.conf.js
+export.config = {
+  // ...
+  services: ['applitools'],
+  applitools: {
+    // options
+    // ...
+  }
+  // ...
+};
+```
+
+## Usage
+
+Once the service is added you just need to call the `browser.check` command to compare images within the badge. The command takes a screenshot name so Applitools can compare it always with the correct image from the baseline, e.g.
```

Comment:
browser.check does not imply what it checks, I think a better name can be used for this to make clear what it does

Reply:
@erwinheitzman you right, what name would you suggest?

Reply:
perhaps something like compareImage(s) / checkImage(s) / matchImage(s) is more appropriate

Reply:
I go with `takeSnapshot`